File: LiberatePDF2-Rest-Service/src/main/java/rocks/huwi/liberatepdf2/restservice/storage/StorageException.java
package rocks.huwi.liberatepdf2.restservice.storage;

public class StorageException extends RuntimeException {

	private static final long serialVersionUID = -4413651601072818152L;

	public StorageException(final String message) {
		super(message);
	}

	public StorageException(final String message, final Throwable cause) {
		super(message, cause);
	}
}

File: LiberatePDF2-Rest-Service/src/main/java/rocks/huwi/liberatepdf2/restservice/storage/StorageException.kt
package rocks.huwi.liberatepdf2.restservice.storage

open class StorageException : RuntimeException {
    constructor(message: String?) : super(message)
    constructor(message: String?, cause: Throwable?) : super(message, cause)

    companion object {
        private const val serialVersionUID = -4413651601072818152L
    }
}